bpython/repl.py

```diff
@@ -601,7 +604,7 @@ def get_args(self):
                 try:
                     fake_cursor = self.current_line.index(func) + len(func)
                     f = simpleeval.evaluate_current_attribute(
-                            fake_cursor, self.current_line, self.interp.locals)
+                        fake_cursor, self.current_line, self.interp.locals)
```

Comment:
I'm not very familiar with these rules, is this recommended by PEP8? I interpret https://www.python.org/dev/peps/pep-0008/#indentation as saying the original is correct.